doc/apiv3-documentation.apib

```diff
@@ -2271,229 +2274,792 @@ Lists the columns available for work packages of this project in the form of fie
+                },
+                "from": {
                     "_links": {
-                        "allowedValues": [
-                            { "href": "/api/v3/statuses/1", "title": "New" },
-                            { "href": "/api/v3/statuses/2", "title": "Closed" }
-                        ]
+                        "allowedValues": {
+                          "href": "/api/v3/work_packages", "title": "Work packages"
+                        }
                     },
 
-                    "name": "Status",
-                    "type": "Status",
+                    "name": "From work package",
+                    "type": "WorkPackage"
+                },
+                "to": {
+                    "_links": {
+                        "allowedValues": {
+                            "href": "/api/v3/work_packages", "title": "Work packages"
+                        }
+                    },
 
-                    "_embedded": {
-                        "allowedValues": [
-                            {
-                                "_links": { "self": { "href": "/api/v3/statuses/1" } },
-                                "_type": "Status",
-                                "id": 1,
-                                "name": "New",
-                                "position": 1,
-                                "isDefault": true,
-                                "isClosed": false,
-                                "defaultDoneRatio": 0,
-                                "createdAt": "2014-05-21T08:51:20Z",
-                                "updatedAt": "2014-05-21T09:12:00Z"
-                            },
-                            {
-                                "_links": { "self": { "href": "/api/v3/statuses/2" } },
-                                "_type": "Status",
-                                "id": 2,
-                                "name": "Closed",
-                                "position": 2,
-                                "isDefault": false,
-                                "isClosed": true,
-                                "defaultDoneRatio": 100,
-                                "createdAt": "2014-05-21T08:51:20Z",
-                                "updatedAt": "2014-05-21T09:12:00Z"
-                            }
-                        ]
-                    }
+                    "name": "To work package",
+                    "type": "WorkPackage"
+                },
+                "delay": {
+                  "name": "Relation delay",
+                  "type": "Integer",
+                  "writable": true
                 }
             }
 
-## view the schema [GET]
-
-This is an example of how a schema might look like. Note that this endpoint does not exist in the actual implementation.
+## View relation schema [GET]
 
 + Response 200 (application/hal+json)
 
-    [Example Schema][]
+    [Relation schema][]
 
-# Group Statuses
+## Relation schema for type [/api/v3/relations/schema/{type}]
```

Comment:
So we keep the `/api/v3/relations/schema` and drop `/api/v3/relations/schema/{type}`?